libcontainer/specconv/spec_linux.go

```diff
@@ -433,6 +434,11 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 				MemBwSchema:   spec.Linux.IntelRdt.MemBwSchema,
 			}
 		}
+
+		// update timens offsets
+		if spec.Linux.TimeOffsets != nil {
+			config.TimeOffsets = spec.Linux.TimeOffsets
+		}
```

Comment:
Here `if` is not needed -- worst case scenario is you assign `nil` to  `config.TimeOffsets`.

(In other places `if` is needed to avoid nil dereferences, but not here).